practice/table-print.py

```diff
@@ -0,0 +1,70 @@
+"""Pretty table print.
```

Comment:
Please try to commit only the practice problem in this branch. Remind yourself what branch you're on using `git branch`.